### DISCUSSION TITLE
RESTORAN BULMA

### DISCUSSION THREAD

**Discussion by TahaGuzell** [sonsuzus/sonsuzus#496] [Soru Cevap]
Tayvan'da kutulu yemekler çok yaygındır ve öğrenciler ve ofis çalışanları için uygun fiyatlı ve besin açısından dengeli seçenekler sunar. Bu yemekler genellikle çeşitli sebzeler, protein ve pirinç içerir ve dengeli bir diyet sağlar. Yerel self-servis restoranlarda çok sayıda seçenek mevcut olduğundan, hızlı ve sağlıklı bir öğle yemeği arayanlar için popüler bir seçimdir.

1'den 5'e kadar numaralandırılmış beş Tayvanlı self-servis restoranı var. Pazartesiden cumaya her gün farklı bir restoranı ziyaret edip öğle yemeği için bir kutu yemek hazırlamayı planlıyorsunuz. Pazartesiden perşembeye kadar her seferinde farklı bir restoranı ziyaret ederek bir restoranı ziyaret ettiniz. Şimdi cuma ve bu hafta henüz gitmediğiniz son kalan restoranı ziyaret etmek istiyorsunuz.

Pazartesiden perşembeye kadar her gün için bir tane olmak üzere daha önce ziyaret ettiğiniz dört restoranı girdi olarak alan ve cuma günü ziyaret etmeniz gereken restoranı çıktı olarak veren bir program yazın.

**Comment by TahaGuzell** [comment]
```
def son_restorani_bul(a,b,c,d):
    tumrest = {1, 2, 3, 4, 5}

    gidilenrest = {a, b, c, d}

    cumarest = tumrest - gidilenrest

    cumarest = list(cumarest)
    print(f"Bu hafta Cuma günü gitmeniz gereken restoran: {cumarest[0]}")

a = int(input("Pazartesi gittiğiniz restoran numarasını girin (1-5): "))
b = int(input("Salı gittiğiniz restoran numarasını girin (1-5): "))
c = int(input("Çarşamba gittiğiniz restoran numarasını girin (1-5): "))
d = int(input("Perşembe gittiğiniz restoran numarasını girin (1-5): "))

son_restorani_bul(a,b,c,d)
```

**Comment by sonsuzus** [comment]
Daha önce çözülen bir problemi çözdüğün için bu sayılmıyor malesef. Yeni soru eklemelisin. Ayrıca kod da hatalı. Girdi ve çıktılara uyum sağla.
